packages/ra-data-graphql/src/index.ts

```diff
@@ -221,15 +221,14 @@ export default async (options: Options): Promise<DataProvider> => {
 };
 
 const handleError = (error: ApolloError) => {
-    console.error({ error });
```

Comment:
I think this console.error should not be here. Since `handleError` re-throws, it should be logged / catched at higher levels.